common/utils/k8_utils.py

```diff
@@ -165,3 +166,72 @@ def ping_and_verify_no_packet_loss(pod_name, dest_ip, count=10):
         return False
     else:
         return True
+
+
+def check_service_status(service_name, expected_status=True):
+    """
+    Util function to check the status of service "service_name", default check expected_status=True, i.e. running
+    params:
+    service_name: String
+
+    return:
+    Boolean True or False
+    """
+    k8_api = K8_DPDK()
+    log.info(f"Checking if service:{service_name} is running or not")
+    ret = k8_api.check_service_running(service_name)
+
+    if ret == expected_status:
+        return True
+
+    return False
+
+
+def execute_iperf_setup(kube_iperf_path):
+    """
+    Util function to execute setup.sh located at kube_iperf_path/steps/
+
+    params:
+    kube_iperf_path: String
+
+    return:
+    Boolean True/False
+    """
+    local = Local()
+    cmd = f"cd {kube_iperf_path}; ./steps/setup.sh"
+
+    ret, _, _ = local.execute_command(cmd)
+
+    log.info(ret)
+
+    if "Server is running" in ret:
+        log.passed("Successfully executed setup.sh")
+        return True
+
+    log.failed("Failed to execute setup.sh")
+    return False
+
```

Comment:
Is there a util for run.sh?

Reply:
not yet, these libs are created for the test case add/remove services. will create these utils as and when required.